src/Keboola/DbExtractor/Extractor/Extractor.php

```diff
@@ -107,24 +128,37 @@ public function createSshTunnel(array $dbConfig)
         return $dbConfig;
     }
 
-    abstract public function createConnection($params);
+    /**
+     * @param array $params
+     * @return \PDO|mixed
+     */
+    abstract public function createConnection(array $params);
 
+    /**
+     * @return void|mixed
```

Comment:
This comment is practically saying "anything is fine". Please consider better typehint.

Reply:
I suppose we could restrict it to void|bool. but some extractors currently return direct PDOStatement|false etc...  and would need to be updated

Reply:
I see, that's probably not meaningful work. So this is fine